src/PublishPort.jsx

```diff
@@ -27,7 +65,13 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
                 }>
                 <TextInput id={id + "-ip-address"}
                         value={item.IP || ''}
-                        onChange={(_event, value) => onChange(idx, 'IP', value)} />
+                        validated={validationFailed?.IP ? "error" : "default"}
+                        onChange={(_event, value) => {
+                            utils.validationClear(validationFailed, "IP", onValidationChange);
+                            utils.validationDebounce(() => onValidationChange(validatePublishPort(value, "IP", validationFailed)));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20231005-125841-15f7a463-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)